src/App.css

```diff
@@ -7,52 +7,76 @@
 
 a {
   text-decoration: none;
+  width: 3.688rem;
+  height: 1rem;
+  margin: 1.125rem 2.563rem 0.688rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.813rem;
+  letter-spacing: 1.9px;
+  color: #121212;
+  font-weight: 500;
+  opacity: 0.5;
+}
+
+.fa-user {
+  color: #0290ff;
+}
+
+.userdiv {
+  margin-right: 18%;
 }
 
 .mainlist {
   display: flex;
```

Comment:
- [ ] You have used `display: flex` 9 times and this falls under the bad practice category where you are repeating yourself. Instead please create a class with a display flex and then use it whenever you need it.